app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -205,11 +207,15 @@ open class FenixApplication : Application() {
      * - https://github.com/mozilla/application-services/blob/master/docs/design/megazords.md
      * - https://mozilla.github.io/application-services/docs/applications/consuming-megazord-libraries.html
      */
-    private fun setupMegazord() {
-        // Note: This must be called as soon as possible
+    private fun setupMegazord(): Deferred<Unit> {
+        // Note: Megazord.init() must be called as soon as possible ...
         Megazord.init()
```

Comment:
FWIW call should be [very cheap](https://github.com/mozilla/application-services/blob/master/megazords/fenix/android/src/main/java/mozilla/appservices/Megazord.kt#L12-L15) -- and becomes completely optional if we do https://github.com/mozilla/application-services/issues/1466, which we'd like to.

Reply:
It is indeed! Thanks :-)